climada/hazard/storm_europe.py

```diff
@@ -199,6 +206,209 @@ def _read_one_nc(self, file_name, centroids):
         ncdf.close()
         return new_haz
 
+
+    def read_cosmoe_file(self, fp_file, run_date, event_date=None,
+                         model_name='COSMO-2E', description=None):
+        """Clear instance and read MeteoSwiss cosmoe weather forecast
+        footprint netcdf files into it. One event is one full day in UCT.
```

Comment:
```suggestion
        footprint netcdf files into it. One event is one full day in UTC.
```

Reply:
update: most of my comments also apply to the DWD ICON methods, but i'm not going to point it all out twice! except this one apparently!

Reply:
yes adapted in both places